src/profile-logic/profile-data.js

```diff
@@ -1332,34 +1332,53 @@ export function getFriendlyThreadName(
   thread: Thread
 ): string {
   let label;
+
   switch (thread.name) {
-    case 'GeckoMain':
-      switch (thread.processType) {
-        case 'default':
-          label = 'Parent Process';
-          break;
-        case 'gpu':
-          label = 'GPU Process';
-          break;
-        case 'tab': {
-          const contentThreads = threads.filter(thread => {
-            return thread.name === 'GeckoMain' && thread.processType === 'tab';
-          });
-          if (contentThreads.length > 1) {
-            const index = 1 + contentThreads.indexOf(thread);
-            label = `Content Process (${index}/${contentThreads.length})`;
-          } else {
-            label = 'Content Process';
+    case 'GeckoMain': {
+      if (thread.hasOwnProperty('processName') && thread.processName) {
```

Comment:
I think the inside of this first branch of the `if` statement could use a brief comment explaining why the `thread.processName` is being used, as opposed to just using the `thread.name`. It might be a bit mysterious if reading this code without the broader context of this changeset.